internal/usergroup/parser.go

```diff
@@ -72,15 +72,15 @@ func parseSubidFile(path, username string) (subIDRanges, error) {
 			return rangeList, fmt.Errorf("Cannot parse subuid/gid information: Format not correct for %s file", path)
 		}
 		if parts[0] == username || username == "ALL" {
-			startid, err := strconv.Atoi(parts[1])
+			startid, err := strconv.ParseUint(parts[1], 10, 32)
 			if err != nil {
```

Comment:
This change is overwritten by the next commit, so we may as well drop it / squash it